### ISSUE TITLE
Automatically generate history of past releases

### ISSUE THREAD

**Issue by jenncoop** [steamclock/coda-changelog#100]
We may be able to pull tags/releases to do this

**Comment by AZielinsky95** [comment]
Yeah unfortunately there's no endpoint for creating tables which is an odd limitation. We can do this manually however, which doesn't seem too bad. You'd just duplicate the table once a release is out and rename it.
